lib/ecto/changeset.ex

```diff
@@ -1464,6 +1464,7 @@ defmodule Ecto.Changeset do
   ## Options
 
     * `:message` - the message on failure, defaults to "can't be blank"
+    * `:trim` - trims value before required check to remove whitespace, defaults to true
```

Comment:
What do you think about?

* `:trim` - boolean, sets whether leading whitespaces are removed before running the validation. Defaults to true

Reply:
Sounds definitely better and more precise.